ddl/column_test.go

```diff
@@ -156,7 +160,9 @@ func (s *testColumnSuite) TestColumn(c *C) {
 	err = ctx.FinishTxn(false)
 	c.Assert(err, IsNil)
 
-	values, err := t.RowWithCols(ctx, h, t.Cols())
+	txn, err = ctx.GetTxn(false)
```

Comment:
Seems needless, in your code here u have not commit the txn, so u can directly use txn in line 145.